### DISCUSSION TITLE
Scoping external styles

### DISCUSSION THREAD

**Discussion by matthewp** [withastro/roadmap#236] [Proposal]
Currently there is no way to get scoping of styles outside of the `<style>` tag. 

Some people prefer to break their CSS into separate files. Presently if you do this and then `import` or `@import` the style it will remain unscoped.

Here's 1 user reporting the desire to do this: https://discord.com/channels/830184174198718474/1001537495890145312

**Comment by jasikpark** [comment]
One option that supports this ideal to a degree is CSS Modules, though it doesn't map anymore since the specificity is different from vanilla Astro scoped styles
